recipes/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
You need to move this into a recipe folder, i.e. `recipes/uarray/meta.yaml`

Please also remove the explanatory comments here.